modules/ROOT/pages/build-application-from-api.adoc

```diff
@@ -0,0 +1,198 @@
+= Build an Application from an API Specification
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: build, application, api, specification
+
+During your app development lifecycle, you can build Mule REST or SOAP APIs from an API specification using APIkit.  You develop APIs based on the following modeling languages:
+
+* RAML (RESTful API Modeling Language)
+* WSDL (Web Service Description Language)
+* OData-Enabled APIs using APIkit OData Extension
+
+By using APIkit in Anypoint Studio, you can simplify the design and implementation of APIs by automating Mule flows creation based on importing either a RAML, or WSDL specification file. After generating the API, you can simulate calls to test the API and then finalize its implementation. +
+
+Additionally, APIkit helps you build applications using Apache Maven, but before you jump into the workflows of building an application using APIkit, learn the basic APIkit anatomy and understand the scaffolding technique that generates Mule flows when you start APIkit projects.
+
+[NOTE]
+MuleSoft recommends Design Center for creating and syncing RAML code; however, Anypoint Studio includes a RAML editor that provides syntax validation and suggestions if you want to perform the entire development within Studio.
+
+== APIkit Anatomy
+A REST or SOAP project that uses APIkit has the following parts:
+
+* A RAML or WSDL interface.
+* A main flow
+* Backend flows
+
+.REST Studio Project.
+image::mruntime-rest-studio-project.png[REST Studio Project]
+
+.SOAP Studio Project.
+image::mruntime-soap-studio-project.png[SOAP Studio Project]
+
+=== API Interface
+The interface defines the API, designating resources that contain or access data and specify actions, such as GET and PUT, on the example data
+
+==== REST Interface
+The relationship between REST resources and actions is called resource-action pairing. The interface exposes internal data assets requested by the REST service.
+
+==== SOAP Interface
+The interface defined by a WSDL file exposes a number of services. Each service has a set of operations. The underlying type of data is XML defined by schemas within the WSDL file or by an imported XSD file.
+
+=== Main Flow
+The main flow for these APIs manages these functions:
+
+* Exposes the API using HTTP or Jetty.
```

Comment:
Exposes the API using the HTTP connector

Reply:
Updated.